src/main/java/com/crowdin/cli/commands/picocli/DownloadSubcommand.java

```diff
@@ -31,6 +33,9 @@ class DownloadSubcommand extends ActCommandWithFiles {
     @CommandLine.Option(names = {"-l", "--language"}, paramLabel = "...", order = -2)
     protected List<String> languageIds;
 
+    @CommandLine.Option(names = {"--exclude-language"}, paramLabel = "...", order = -2)
```

Comment:
let's add the alias:

```suggestion
    @CommandLine.Option(names = {"-el", "--exclude-language"}, paramLabel = "...", order = -2)
```